src/lib/merkle_ledger_tests/test_stubs.ml

```diff
@@ -1,5 +1,82 @@
 open Core
+open Unsigned
 module Account = Coda_base.Account
+
+module Balance = struct
+  include Currency.Balance
```

Comment:
I'm weary of letting through a monkey-patch since we've had so many problems with this type of code before -- but it seems okay since this is only in a test stub.

I do think it would be clearer for future readers of the code if instead of monkey-patching the real Currency.Balance to support the UInt64 APIs, we defined the test stubs in such a way to conform to the balance interface (feel free to add `one` to currency's signatures though). Basically can we instead shadow `add` to also return an option.

Reply:
I added `one` to the interface for `Currency.Balance`. 

The addition of balances wasn't relevant to the one test where it was used, so I just changed the test.

Reply:
Oh, there's another test where `add` is used. Looking at that.

Reply:
I was able to do the sum in `Int` for that particular test, in a way that can't overflow.